### DISCUSSION TITLE
Floris Workshop 2019 recording

### DISCUSSION THREAD

**Discussion by matheusseidel** [NatLabRockies/floris#602] [General]
Hello, 

Does anyone have the recording of the 2019 Floris Workshop where the fallowing slides were presented?
[https://www.nrel.gov/wind/assets/pdfs/systems-engineering-workshop-2019-floris.pdf](url)

Thanks.

**Comment by rafmudaf** [comment]
@bayc Was this your workshop? The link above is broken, so if you have the slides we can host the on the FLORIS docs.

**Reply by christiannvaughn** [reply]
I don't think there is a recording.
